### PR TITLE
Completed layout for teacher and student registration page as well as worked on login page. 

### PR DESCRIPTION
**Pull request by Acr0batOG**
The pages don't have functionality yet such as a database query or password regex checking. But the layout is complete and the hrefs should all work. Let me know what you think of the student and teacher registration pages and login layout. I mostly went off of Ankitts design from the first page. We can work on other details and additions to the page soon.

### CODE REVIEW

File: eduquest/app/Register/page.tsx [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+      <div className="text-center text-3xl lg:text-3xl lg:mx-4 sm:mx-2">
+        Are you a
+      </div>
+      <div className="grid text-center space-y-4 sm:space-y-0 lg:space-y-0  lg:grid-cols-2 sm:grid-cols-2 mt-6">
+        <Link href="/Register/TeacherRegistration">
+          <h1 className="text-3xl lg:text-4xl lg:mx-4 sm:mx-2 bg-red-600 p-4 rounded-md ">
+            Teacher
+          </h1>
+        </Link>
+        <Link href="/Register/StudentRegistration">
```
Comment (ankkprak) [inline eduquest/app/Register/page.tsx:21]: Moved Teacher and Student registration into registration parent folder

File: eduquest/app/Register/TeacherRegistration/page.tsx [hunk 2]
```diff
@@ -0,0 +1,55 @@
+import Image from "next/image";
+import Link from "next/link";
+
+export default function TeacherRegistration() {
+  return (
+    <main className="px-4 bg-gradient-to-b from-amber-100 to-amber-500 min-h-screen flex items-center flex-col">
```
Comment (ankkprak) [inline eduquest/app/Register/TeacherRegistration/page.tsx:6]: We'll need to create form components, since this is getting a bit messy to read.

**Comment by ankkprak** [comment]
Removed HTML  styling tags and replaced with equivalent CSS. Fixed a few grammar issues. Changed file directory to move the two kinds of registration into the registration folder. Rounded some buttons.

**Comment by ankkprak** [comment]
We could merge this now to show the TA, or we could figure out how to do authentication first and then merge it.

**Review by nolannish — COMMENTED** [review]
Changes that were requested seem to have been made.

**Pull request merged**